com/ArchiveService.py

```diff
@@ -506,7 +617,8 @@ def fill_event(ev, desc = None):
         retry_folder = retry_folder.replace('%reason%', 'rinex_issues')
 
         # add more verbose output
-        fill_event(e.event, '\n%s: (file moved to %s)' % (os.path.relpath(crinez, Config.repository_data_in),
+        fill_event(e.event, '\n%s: (file moved to %s)' % (os.path.relpath(
+            crinez, Config.repository_data_in),
```

Comment:
inconsistent indentation here-- line 622 is aligned to the opening parens as expected, but 621 is 4 space indented relative to the parent line. I believe that this would be a valid alternative:

```python
        fill_event(e.event, '\n%s: (file moved to %s)'
                   % (os.path.relpath(crinez, Config.repository_data_in),
```